model/Core/Individuals/SpdxOrganization.md

```diff
@@ -0,0 +1,21 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SpdxOrganization
+
+## Summary
+
+An Individual Value for Organization representing the SPDX Project.
+
+## Description
+
+SpdxOrganization is used by named individuals for their creationInfo.
+
+## Metadata
+
+- name: SpdxOrganization
+- type: Organization
+
+## Property Values
+
+- spdxId: https://spdx.org/
```

Comment:
No, there is no `spdxId` property.

Use `IRI` in Metadata as in https://github.com/spdx/spdx-3-model/blob/main/model/ExpandedLicensing/Individuals/NoneLicense.md